etc/ProdOfflineConfiguration.py

```diff
@@ -223,7 +223,7 @@
            archival_node="T0_CH_CERN_MSS",
            tape_node="T1_US_FNAL_MSS",
            disk_node="T1_US_FNAL_Disk",
-           raw_to_disk=False,
+           raw_to_disk=True,
```

Comment:
We should not change the config for the default dataset.
```suggestion
           raw_to_disk=False,
```
otherwise, we would need to add `raw_to_disk=False` for every dataset. Makes more sense to keep it False and change the few we need in disk

Reply:
Also, we don't want undefined datasets (like those that come from minidaq runs) to go to disk by default.